server/protocols/webadmin/webadmin-jmap/src/test/java/org/apache/james/webadmin/data/jmap/UserIdentitiesRoutesTest.java

```diff
@@ -0,0 +1,293 @@
+            Optional.of("htmlSignature 1"));
+
+        // identity: custom
+        Mono.from(identityRepository.save(BOB, creationRequest)).block();
+
+        String response = when()
+            .get(String.format(GET_IDENTITIES_USERS_PATH, BOB.asString()))
+        .then()
+            .statusCode(HttpStatus.OK_200)
+            .contentType(io.restassured.http.ContentType.JSON)
+            .extract()
+            .body()
+            .asString();
+
+        String expectedResponse = "[" +
+            "    {" +
+            "        \"name\": \"identity name 1\"," +
+            "        \"email\": \"bob@domain.tld\"," +
+            "        \"id\": \"${json-unit.ignore}\"," +
+            "        \"mayDelete\": true," +
+            "        \"textSignature\": \"textSignature 1\"," +
+            "        \"htmlSignature\": \"htmlSignature 1\"," +
+            "        \"sortOrder\": 1," +
+            "        \"bcc\": [" +
+            "            {" +
+            "                \"name\": null," +
+            "                \"email\": \"bcc1@james.org\"" +
+            "            }," +
+            "            {" +
+            "                \"name\": null," +
+            "                \"email\": \"bcc2@james.org\"" +
+            "            }" +
+            "        ]," +
+            "        \"replyTo\": [" +
+            "            {" +
+            "                \"name\": null," +
+            "                \"email\": \"replyTo1@james.org\"" +
+            "            }," +
+            "            {" +
+            "                \"name\": null," +
+            "                \"email\": \"replyTo2@james.org\"" +
+            "            }" +
+            "        ]" +
+            "    }," +
+            "    {" +
+            "        \"name\": \"base name\"," +
+            "        \"email\": \"bob@domain.tld\"," +
+            "        \"id\": \"${json-unit.ignore}\"," +
+            "        \"mayDelete\": false," +
+            "        \"textSignature\": \"text signature base\"," +
+            "        \"htmlSignature\": \"html signature base\"," +
+            "        \"sortOrder\": 100," +
+            "        \"bcc\": [" +
+            "            {" +
+            "                \"name\": \"My Boss bcc 1\"," +
+            "                \"email\": \"boss_bcc_1@domain.tld\"" +
+            "            }" +
+            "        ]," +
+            "        \"replyTo\": [" +
+            "            {" +
+            "                \"name\": \"My Boss 1\"," +
+            "                \"email\": \"boss1@domain.tld\"" +
+            "            }" +
+            "        ]" +
+            "    }" +
+            "]";
+        assertThatJson(response)
+            .when(Option.IGNORING_ARRAY_ORDER)
+            .isEqualTo(expectedResponse);
+    }
+
+    @Test
+    void listIdentitiesShouldSupportDefaultParam() throws Exception {
```

Comment:
How about a test to ensure that GIVEN some user identities with different sortOrder, only the highest priority identity is returned?

Reply:
> highest priority

the ticket writes down: `smallest integer`, not `highest`

> How about a test to ensure that GIVEN some user identities with different sortOrder

in this test, I use 2 creation requests: one sortOrder =1, and one sortOrder =2 ,
then the default identity return entry with sortOrder=1,
IMO it is enough

Reply:
Ok I see, i read it quickly...
Maybe variablize the sortOrder to e.g lowPriority, highPriority instead of Optional.of would be more readable.